deploy/teardown_resources.sh

```diff
@@ -7,31 +7,59 @@ set -euxo pipefail
 set_namespace $CONJUR_NAMESPACE_NAME
 
 configure_cli_pod
+
+# Helm Chart clean-up
+rm -f conjur.pem
+
 if [ "${DEV}" = "false" ]; then
-  $cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
+  pushd ../../../secrets-provider/ci
+else
+  pushd ../secrets-provider/ci
+fi
+    find . -type f ! -name '*template.yaml' -delete
+popd
+
+# Delete Helm Chart if already exists
+set_namespace $APP_NAMESPACE_NAME
+if [ "$(helm ls -aq | wc -l | tr -d ' ')" != 0 ]; then
+  helm delete $(helm ls -aq)
 fi
 
+set_namespace $CONJUR_NAMESPACE_NAME
+
+$cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
```

Comment:
why do we need this?

Reply:
this resets the secret value across test runs. A few of our tests update test_secret value and so before the next run, this teardown script reset the value to it's original value